setup.py

```diff
@@ -82,6 +129,7 @@ def package_files(directory):
         "ert-storage >= 0.3.11",
         "fastapi==0.70.1",
         "graphlib_backport; python_version < '3.9'",
+        "grpcio-tools",
```

Comment:
move to dev-requirements.txt as it is not strictly needed for an installation, but rather if you want to change the generated one (which is generated at build/install time).